contrib/init/systemd/docker.service

```diff
@@ -18,8 +14,6 @@ LimitCORE=infinity
 # Only systemd 226 and above support this version.
 #TasksMax=infinity
 TimeoutStartSec=0
-# set delegate yes so that systemd does not reset the cgroups of docker containers
-Delegate=yes
```

Comment:
Right, because Docker might start containerd (we don't package it that way). I'll update the PR.